data.json

```diff
@@ -80,6 +80,15 @@
     "dateAdded": "2018-02-18T18:44:53.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "CodingBlocks",
+    "difficulty": "medium",
+    "description": "Have one merged PR on any of <a href=\"https://cb.lk/gh\">Coding Blocks open source projects</a> and get t-shirt with stickers or even a bag (1 PR - t-shirt, 5+ PRs - bag)",
+    "reference": "https://blog.codingblocks.com/2019/cb-hacktoberfest-2019/",
+    "image": "https://blog.codingblocks.com/content/images/size/w2000/2019/10/asc.jpg",
```

Comment:
I suggest using only the left side of the image. Something like this: https://i.imgur.com/0SrSsNM.png

Reply:
Do you wan to use imgur as source for image? Looks like not reliable source of information for me :) However every website could remove image used in data.json, so maybe imgur is even more reliable :)

Reply:
From my experience, imgur is one of the most robust host. Anyway, we download the image and serve it ourselves.

Reply:
Got it. By "not reliable" I meant you can always clean up your imgur images and accidentally remove this nice image :)
Changed the image with following commit btw.

Reply:
I always create them in non-logged mode, so I'm sure this won't happen :D

Reply:
It looks nice, thanks!